### PR TITLE
Fix docstrings after major refactor

### PR DESCRIPTION
**Pull request by adason**
# Peer Review Information

The module is moved after refactoring. This change fixes docstring by running `make apidoc` to generate and fix rst files in the docs folder. 

# Pull Request Check List

- [ ] Added **tests** for changed code.
- [x] Updated **documentation** for changed code.

### CODE REVIEW

File: docs/source/datasetinsights.commands.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
+datasetinsights.commands
```
Comment (BlairLee) [inline docs/source/datasetinsights.commands.rst:1]: Don't we do `make clean` before we merge to master?

File: docs/Makefile [hunk 2]
```diff
@@ -15,7 +15,7 @@ clean:
 	find $(SOURCEDIR) -name "*.rst" | grep -v index | xargs rm -f
 
 apidoc:
-	sphinx-apidoc -f --templatedir=$(TEMPLATEDIR) -o $(SOURCEDIR) ../datasetinsights/ ../datasetinsights/visualization ../datasetinsights/dashboard.py ../datasetinsights/data/datasets/protos
+	sphinx-apidoc -f --templatedir=$(TEMPLATEDIR) -o $(SOURCEDIR) ../datasetinsights/ ../datasetinsights/ststs/visualization ../datasetinsights/dashboard.py ../datasetinsights/datasets/protos
 html: apidoc
 	sphinx-build -b html $(SOURCEDIR) $(BUILDDIR)/html
 	@echo
```
Comment (BlairLee) [inline docs/Makefile:21]: When I track this branch, and do `make html`, why do I get this error:
```
Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/Users/bowen.li/miniconda3/envs/bowen/lib/python3.7/site-packages/sphinx/config.py", line 319, in eval_config_file
    execfile_(filename, namespace)
  File "/Users/bowen.li/miniconda3/envs/bowen/lib/python3.7/site-packages/sphinx/util/pycompat.py", line 88, in execfile_
    exec(code, _globals)
  File "/Users/bowen.li/Documents/dataset-insights/docs/source/conf.py", line 21, in <module>
    release = pkg_resources.get_distribution(project).version
  File "/Users/bowen.li/miniconda3/envs/bowen/lib/python3.7/site-packages/pkg_resources/__init__.py", line 482, in get_distribution
    dist = get_provider(dist)
  File "/Users/bowen.li/miniconda3/envs/bowen/lib/python3.7/site-packages/pkg_resources/__init__.py", line 358, in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
  File "/Users/bowen.li/miniconda3/envs/bowen/lib/python3.7/site-packages/pkg_resources/__init__.py", line 901, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/Users/bowen.li/miniconda3/envs/bowen/lib/python3.7/site-packages/pkg_resources/__init__.py", line 787, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'datasetinsights' distribution was not found and is required by the application

make: *** [html] Error 2
```

File: docs/source/datasetinsights.commands.rst (hunk 1 above)
Comment (adason) [reply docs/source/datasetinsights.commands.rst:1]: make clean is not required as html files are ignored. RST files has to be checked in for readthedocs to build automatically.

File: docs/Makefile (hunk 2 above)
Comment (adason) [reply docs/Makefile:21]: You need to run `poetry install` to install datasetinsights.

File: docs/Makefile [hunk 3]
```diff
@@ -15,7 +15,7 @@ clean:
 	find $(SOURCEDIR) -name "*.rst" | grep -v index | xargs rm -f
 
 apidoc:
-	sphinx-apidoc -f --templatedir=$(TEMPLATEDIR) -o $(SOURCEDIR) ../datasetinsights/ ../datasetinsights/visualization ../datasetinsights/dashboard.py ../datasetinsights/data/datasets/protos
+	sphinx-apidoc -f --templatedir=$(TEMPLATEDIR) -o $(SOURCEDIR) ../datasetinsights/ ../datasetinsights/ststs/visualization ../datasetinsights/dashboard.py ../datasetinsights/datasets/protos
 html: apidoc
```
Comment (sanjayuconn) [inline docs/Makefile:19]: datasetinsights/ststs/visualization: typo in ststs

File: docs/source/datasetinsights.evaluation_metrics.rst [hunk 4]
```diff
@@ -18,10 +18,10 @@ datasetinsights.evaluation\_metrics.average\_precision
...
    :show-inheritance:
 
-datasetinsights.evaluation\_metrics.average\_precision\_2d\_bbox
-----------------------------------------------------------------
+datasetinsights.evaluation\_metrics.average\_precision\_2d
+----------------------------------------------------------
 
-.. automodule:: datasetinsights.evaluation_metrics.average_precision_2d_bbox
+.. automodule:: datasetinsights.evaluation_metrics.average_precision_2d
    :members:
```
Comment (sanjayuconn) [inline docs/source/datasetinsights.evaluation_metrics.rst:25]: Have you renamed the module from average_precision_2d_bbox to average_precision_2d? btw what's the reason for this change?

File: docs/Makefile (hunk 2 above)
Comment (BlairLee) [reply docs/Makefile:21]: Got it.

File: docs/source/datasetinsights.evaluation_metrics.rst (hunk 4 above)
Comment (adason) [reply docs/source/datasetinsights.evaluation_metrics.rst:25]: It was changed as part of this [PR](https://github.com/Unity-Technologies/datasetinsights/pull/20). The docs was not updated accordingly in that PR.

File: docs/Makefile (hunk 3 above)
Comment (adason) [reply docs/Makefile:19]: Thanks for catching this. It's been fixed now.

**Review by sanjayuconn — APPROVED** [review]
